deployment/docker/kafka-puller/run_app.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+bootstrap_server="$1"
+topic_name="$2"
+./gradlew build
+
+while true
+do
+  nc -v -l -p 9220 |  java -cp build/libs/KafkaPrinter.jar:"$(ls extracted/*/*/*.jar | tr \\n :)" org.opensearch.KafkaPrinter -o "$target_endpoint" "$topic_name"
```

Comment:
Any issue running this java command locally? It would probably be good to have a comment describing what this java command is doing and maybe even better to store the result of the command to get the jars in a variable to use so that this line is a bit simpler

Reply:
No issue running this locally. Great point, I'll add comments and store the result of the `ls` command.